### ISSUE TITLE
`internal` is default access control level

### ISSUE THREAD

**Issue by bzmario** [shogo4405/lf.swift#105]
`internal` is default level of access control, so there's no need to declare any type as `internal` in `lf.swift` module explicitly.

**Comment by shogo4405** [comment]
Yes. I'm sure.
But I think need `open internal(set)` `public internal(set)`. don't you?

**Comment by bzmario** [comment]
I agree with that. But declarations like:

```swift
final internal class RTMPSocket: NetSocket, RTMPSocketCompatible {
...
```

I think it's unnecessary.

**Comment by shogo4405** [comment]
oops :) Thank u. I will remove 0.5.0 and 0.4.2

**Comment by shogo4405** [comment]
fixed 0.5.0.
